src/main/models.ts

```diff
@@ -39,6 +38,8 @@ export type SignupProfile = {
   liteOnly?: boolean
 }
 
+export type SignupProfile = SignupProfileData & { password: string }
```

Comment:
Perhaps we can make the password flow more explicit in this signup type

Reply:
Sorry, I didn't understand what you mean.

Reply:
`SignupProfile` as a type name doesn't convey the fact that's it's a password signup flow.

Reply:
I think that renaming would be breaking-change since the type is exported and already used.

Reply:
Good point ! 😄